ui/app/templates/vault/cluster.hbs

```diff
@@ -3,37 +3,45 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 <Sidebar::Nav::Cluster />
-{{#if this.auth.currentToken}}
-  {{#if this.customMessages.bannerMessages.length}}
-    {{#each this.customMessages.bannerMessages as |bannerMessage|}}
-      <Hds::Alert @type="inline" @color="warning" data-test-alert={{bannerMessage.id}} as |A|>
-        <A.Title data-test-alert-title={{bannerMessage.id}}>{{bannerMessage.title}}</A.Title>
-        <A.Description data-test-alert-description={{bannerMessage.id}}>
-          {{bannerMessage.message}}
-        </A.Description>
-        <A.Description class="has-top-margin-xs">
-          {{! TODO: VAULT-22908 display links when api is updated to { link: { 'learn': 'www.learn.com'} } }}
-        </A.Description>
-      </Hds::Alert>
-    {{/each}}
+{{#each this.customMessages.bannerMessages as |bannerMessage|}}
+  {{#if (get this.customMessages.bannerDismissObject bannerMessage.id)}}
+    <Hds::Alert
+      @type="inline"
+      @color="highlight"
+      data-test-alert={{bannerMessage.id}}
```

Comment:
It might be useful for these selectors to be specific to customMessages since they'll likely show at the same time as other ui alert banners
```suggestion
      data-test-custom-alert={{bannerMessage.id}}